### ISSUE TITLE
[DOC]: Some urls are not correct in Developer's Guide

### ISSUE THREAD

**Issue by Q-Bug4** [Netflix/conductor#3803]
## What are you missing in the docs
In [Building Conductor From Source](https://conductor.netflix.com/devguide/running/source.html), some urls are start with `http://http://` while the href link is not right. For example below:
![image](https://github.com/Netflix/conductor/assets/48374300/c4eaf437-a645-46f2-ae67-0122172eebbb)

**Comment by rohan-kapse** [comment]
hey, do u expect to change the urls from html file right?

**Comment by Q-Bug4** [comment]
> hey, do u expect to change the urls from html file right?

Yes. But I had checked the markdown files, the urls are right. Maybe there are something wrong in which convert markdown into html.

**Comment by rohan-kapse** [comment]
@Q-Bug4 isn't there any update from the developer team of this project so, maybe we can help?
